app.js

```diff
@@ -1,47 +1,147 @@
 console.log('Starting a project!')
 
 const secondsElement = document.getElementById('seconds');
-const minutesElement = document.getElementById('minutes');
-const hoursElement = document.getElementById('hours');
+const milisecondsElement = document.getElementById('miliseconds');
 const startAndResetButtonElement = document.getElementById('start');
 const boardElement = document.getElementById('board');
+let scoreBoardElement = document.getElementById('score-board');
+let lifeBoardElement = document.getElementById('lives')
+let score = 0;
 let intervalId;
-let totalseconds = 0;
+let totalmiliseconds = 300;
 let isTimerRunning = false;
+let life = 3;
+let width = 50;
+let height = 50;
 
 function pad(val){
   const value = val + "";
-  if(value.length < 2){
+  if (value.length < 2) {
     return "0" + value;
   } else {
     return value;
   }
 }
 
-function setTime(){
-  ++totalseconds;
-  secondsElement.innerHTML = pad(totalseconds) - (minutesElement.innerHTML*60) - (hoursElement.innerHTML*3600);
-  minutesElement.innerHTML = pad(parseInt(totalseconds/60)) - (hoursElement.innerHTML*60);
-  hoursElement.innerHTML = pad(parseInt(totalseconds/3600));      
+function createTarget() {
+  const boardParams = boardElement.getBoundingClientRect();
+  let x=Math.random()*(boardParams.width - 50);
+  x=Math.round(x);
+  let y=Math.random()*(boardParams.height - 50);
+  y=Math.round(y);
+  let paragrath = document.createElement('p');
+  paragrath.classList.add('target');
+  paragrath.style.left = x + 'px';
+  paragrath.style.top = y + 'px';
+  paragrath.addEventListener('mouseover', onHoveringTarget);
+  paragrath.addEventListener('click', onClickTarget);
+  boardElement.append(paragrath);
 }
 
-function startTheGame(){
+function deleteTarget(){
+  const targetElement = document.getElementsByClassName('target');
```

Comment:
this is not correct as it always delete the first target of the array...

if you have 3 targets and player click on the 3 one the function will remove the first ...